### DISCUSSION TITLE
An option how to install etcd

### DISCUSSION THREAD

**Discussion by AkakievKD** [kubesphere/kubekey#1087] [Ideas]
Hello, Can you add an option to kubekey how to install etcd, in container or host

**Comment by FeynmanZhou** [comment]
> Hello, my name is Konstantin. I am DevOps. Can you add an option to kubekey how to install etcd, in container or host

Hi @Nello-Angelo ,

By default, KubeKey will help you to install etcd in All-on-one installation. If you are bootstrapping a multi-node cluster, please refer to this [guide](https://kubesphere.io/docs/installing-on-linux/introduction/multioverview/).

![image](https://user-images.githubusercontent.com/40452856/154839253-efd1a76e-ecc7-4b16-aac2-4a7d72bc3069.png)

**Reply by AkakievKD** [reply]
It does not help. Can kubekey install etcd in container ? Can i say kubekey "to install etcd in container" in configuration of the config-sample.yaml or using special keys ? Yes or No ?

**Comment by pixiake** [comment]
At present, it is not supported to choose the installation method of etcd. 
The next version will support selecting the installation method of etcd, which will include creating etcd with kubeadm, in which the etcd exists as container and static pod.
